tools/analyzer_plugin/lib/src/plugin.dart

```diff
@@ -95,9 +95,11 @@ class OverReactAnalyzerPlugin extends ServerPlugin
       ..performanceLog = performanceLog
       ..fileContentOverlay = fileContentOverlay;
     final result = contextBuilder.buildDriver(root);
-    runZonedGuarded(() {
+    runZoned(() {
       result.results.listen(processDiagnosticsForResult);
-    }, (e, stackTrace) {
+      // ignore: avoid_types_on_closure_parameters
+      // TODO: Once we are ready to bump the SDK lower bound to 2.8.x, we should swap this out for `runZoneGuarded`.
```

Comment:
There can't be any other comments between the ignore and the targeted line; this still lints for me locally
```suggestion
      // TODO: Once we are ready to bump the SDK lower bound to 2.8.x, we should swap this out for `runZoneGuarded`.
      // ignore: avoid_types_on_closure_parameters
```